### PR TITLE
WIP - Bump API docs for config.openshift.io

### PR DESCRIPTION
**Pull request by adambkaplan**

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -292,12 +294,12 @@ imports:
...
   - util/retry
 - name: k8s.io/component-base
-  version: f0322db00a103489b85cb3835886741a3911636b
+  version: f3799749b6b739bf4bb6ef1c79e63081bf109f15
   subpackages:
   - cli/flag
   - logs
 - name: k8s.io/klog
-  version: 8e90cee79f823779174776412c13478955131846
+  version: cd60aa438f9c750641d5feacd2999e58df8af28f
```
Comment (adambkaplan) [inline glide.lock]: Somehow this version of klog is not compatible with the vendored k8s.io/client-go. How did this happen?

**Comment by ironcladlou** [comment]
This fixes https://bugzilla.redhat.com/show_bug.cgi?id=1738401

**Comment by adambkaplan** [comment]
/close

Replaced by #87

**Pull request closed without merging**